arangod/Aql/ExecutionBlockImpl.cpp

```diff
@@ -897,6 +900,7 @@ template class ::arangodb::aql::ExecutionBlockImpl<SubqueryStartExecutor>;
 template class ::arangodb::aql::ExecutionBlockImpl<TraversalExecutor>;
 template class ::arangodb::aql::ExecutionBlockImpl<SortingGatherExecutor>;
 template class ::arangodb::aql::ExecutionBlockImpl<UnsortedGatherExecutor>;
+template class ::arangodb::aql::ExecutionBlockImpl<UnsortingGatherExecutor>;
```

Comment:
This naming will likely confuse the bajeezus out of me....

Reply:
We must rename one of those. I'd prefer renaming the `UnsortedGatherExecutor` to something like `ParallelUnsortingGatherExecutor`. Imho we could do it in another PR.
WDYT?

Reply:
We can do this in another PR, but we should not forget (aka put this in the backlog); Not quite sure what `Unsorting` stands for, but this is a naming nitpick, I  assume it means "it's not sorting" as opposed to the `SortingGather` which does a distribute (requesting sorted results) & mergesort after,

Reply:
> I assume it means "it's not sorting" as opposed to the SortingGather

Yes.

I'll do it in the cleanup PR.